Ransomwares/Hive/Impair Defenses/disable_security_software.md

```diff
@@ -0,0 +1,52 @@
+# Hive
+
+Use this query to look for ransomware Hive behavior in the environment.
+
+## Query
+
+Search for Windows Defender's features disabling events.
+
+```
+type_id:8001 AND operation:1 AND process.cmd_line:"reg.exe add \"hklm\\software\\policies\\microsoft\\windows defender" AND process.cmd_line:["disableantispyware\" /t reg_dword /d \"1\"" OR "disableantivirus\" /t reg_dword /d \"1\"" OR "mpenablepus\" /t reg_dword /d \"0\"" OR "disablebehaviormonitoring\" /t reg_dword /d \"1\"" OR "disableioavprotection\" /t reg_dword /d \"1\"" OR "disableonaccessprotection\" /t reg_dword /d \"1\"" OR "disablerealtimemonitoring\" /t reg_dword /d \"1\"" OR "disablescanonrealtimeenable\" /t reg_dword /d \"1\"" OR "disableenhancednotifications\" /t reg_dword /d \"1\"" OR "disableblockatfirstseen\" /t reg_dword /d \"1\"" OR "spynetreporting\" /t reg_dword /d \"0\"" OR "submitsamplesconsent\" /t reg_dword /d \"0\""]
```

Comment:
Could you please add actor/target process name so that this query wont match every process command lines

Reply:
The actor here is the sample that was executed. I'm unsure what approach to use here if the sample's name is not static. Should I just remove this query?

Reply:
i checked the events in SEDR and the executed sample's name is the only process name that's associated with the `reg.exe add` process.cmd_line events.

But the query above will only match `reg.exe` events, it's interpreted as process.cmd_line field should contain `reg.exe add "hklm\software\policies\microsoft\windows defender` string and should also contain one of the following strings:
- disableantispyware" /t reg_dword /d "1"
- disableantivirus" /t reg_dword /d "1"
- mpenablepus" /t reg_dword /d "0"
- disablebehaviormonitoring" /t reg_dword /d "1"
- disableioavprotection" /t reg_dword /d "1"
- disableonaccessprotection" /t reg_dword /d "1"
- disablerealtimemonitoring" /t reg_dword /d "1"
- disablescanonrealtimeenable" /t reg_dword /d "1"
- disableenhancednotifications" /t reg_dword /d "1"
- disableblockatfirstseen" /t reg_dword /d "1"
- spynetreporting" /t reg_dword /d "0"
- submitsamplesconsent" /t reg_dword /d "0"